src/bynder-js-sdk.js

```diff
@@ -597,4 +597,86 @@ export default class Bynder {
         );
         return request.send();
     }
+
+    /**
+     * Create the collection information according to the name provided.
+     * @see {@link http://docs.bynder.apiary.io/#reference/collections/specific-collection-operations/create-collection|API Call}
+     * @param {Object} queryObject={} - An object containing the id of the desired collection.
+     * @param {String} queryObject.name - The name of the desired collection.
+     * @param {String} queryObject.description - The description of the desired collection.
+     * @return {Promise} Collection - Returns a Promise that, when fulfilled, will either return an Object with the
+     * collection or an Error with the problem.
+     */
+    createCollection(queryObject) {
+        if (!this.validURL()) {
+            return rejectURL();
+        }
+        if (!queryObject.name) {
+            return rejectValidation('collection', 'name');
+        }
+        const request = new APICall(
+            this.baseURL,
+            'v4/collections/',
+            'POST',
+            this.consumerToken,
+            this.accessToken,
+            queryObject,
+        );
+        return request.send();
+    }
+
+    /**
+     * Share the collection to the recipients provided.
+     * @see {@link http://docs.bynder.apiary.io/#reference/collections/specific-collection-operations/share-collection|API Call}
+     * @param {Object} queryObject={} - An object containing the id of the desired collection.
+     * @param {String} queryObject.id - The id of the shared collection.
+     * @param {String} queryObject.recipients - The email addressed of the recipients.
+     * @param {String} queryObject.collectionOptions - The recipent right of the shared collection: view, edit
+     * @return {Promise} Collection - Returns a Promise that, when fulfilled, will either return an Object with the
+     * collection or an Error with the problem.
+     */
+    shareCollection(queryObject) {
+        if (!this.validURL()) {
+            return rejectURL();
+        }
+        if (!queryObject.id || !queryObject.recipients || !queryObject.collectionOptions) {
```

Comment:
I think if you put it like this it will make it more objective and pragmatic, no?
```
if (!queryObject.id) {
    return rejectValidation('collection', 'id');
}
if (!queryObject.recipients) {
    return rejectValidation('collection', 'recipients');
}
if (!queryObject. collectionOptions) {
    return rejectValidation('collection', 'collectionOptions');
}
```